protos/sigstore_common.proto

```diff
@@ -169,9 +169,16 @@ message SubjectAlternativeName {
 message X509CertificateChain {
         // The chain of certificates, with indices 0 to n.
         // The first certificate in the array must be the leaf
-        // certificate used for signing. Any intermediate certificates
-        // must be stored as offset 1 to n-1, and the root certificate at
-        // position n.
+        // certificate used for signing.
+        //
+        // Signers MUST NOT include their root CA or CAs in their embedded
```

Comment:
nit, CA -> CA certificates here and below (let's just use the plural)

Reply:
Done!